wai-handler-launch/ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+## 3.0.2.5
```

Comment:
Since this adds a new function, this should be version 3.0.3

Reply:
Bumped it accordingly. Somehow forgot to follow semantic versioning. Force-pushed a fix.